### PR TITLE
tower of hanoi in c

### PR DESCRIPTION
**Pull request by gunkiratk**

### CODE REVIEW

**Comment by t2013anurag** [comment]
@gunkiratk - can you add a readme as well explaining how code works

File: recursion/toh.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+ 
+int main()
+{
+    int n;
+    printf("%s\n","Enter number of disks" );
+    scanf("%d",&n);
+    towerOfHanoi(n, 'A', 'C', 'B');  // A, B and C are names of rods
+    return 0;
+}
```
Comment (t2013anurag) [inline recursion/toh.c]: add a new line.

File: recursion/toh.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+ 
+void towerOfHanoi(int n, char from_rod, char to_rod, char aux_rod)
```
[the remark this answers is not in the document]
Comment (gunkiratk) [reply recursion/toh.c]: There is no cpp version in the directory

[the remark this answers is not in the document]
Comment (t2013anurag) [reply recursion/toh.c]: https://github.com/hacktoberfest17/programming/blob/master/recursion/tower_of_hanoi.java

My bad the java version

**Comment by gunkiratk** [comment]
#205 
Variables changed

**Pull request merged**